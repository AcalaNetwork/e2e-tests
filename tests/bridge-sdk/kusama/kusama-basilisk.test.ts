import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'kusama',
  to: 'basilisk',
  token: 'KSM',
  ignoreFee: true,
})
