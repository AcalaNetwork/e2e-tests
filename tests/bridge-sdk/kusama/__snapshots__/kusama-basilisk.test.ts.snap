// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'kusama' to 'basilisk' using bridgeSDK cross-chain 'KSM' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 8)",
  "toChain": "(rounded 1000)",
}
`;

exports[`'kusama' to 'basilisk' using bridgeSDK cross-chain 'KSM' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 1000,
}
`;
